CHANGELOG.md

```diff
@@ -22,6 +23,49 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+* ### ALL
+    * #### Added
+    * #### Changed
+    * #### Removed
+* ### NI-DCPower
```

Comment:
Pre-existing, but I think we should use the module name here. Something like:

```
* ### `nidcpower` (NI-DCPower)
```

Can we make this change in a separate PR?

Reply:
Agreed.

Changed in https://github.com/ni/nimi-python/pull/1470.